lib/features/modeling/util/ModelingUtil.js

```diff
@@ -37,19 +42,42 @@ export function getParent(element, anyType) {
  * Determines if the local modeling direction is vertical or horizontal.
  *
  * @param {Element} element
+ * @param {ElementRegistry} elementRegistry
  *
  * @return {boolean} false for vertical pools, lanes and their children. true otherwise
  */
-export function isDirectionHorizontal(element) {
+export function isDirectionHorizontal(element, elementRegistry) {
 
-  var types = [ 'bpmn:Participant', 'bpmn:Lane' ];
+  var parent = getParent(element, 'bpmn:Process');
+  if (parent) {
+    return true;
+  }
 
-  var parent = getParent(element, types);
+  parent = getParent(element, 'bpmn:Participant');
   if (parent) {
     return isHorizontal(parent);
-  } else if (isAny(element, types)) {
+  } else if (is(element, 'bpmn:Participant')) {
```

Comment:
Does it mean we don't care about vertical/horizontal Lane anymore?

Reply:
Good catch. We still care.